js/templates/ratingStars.html

```diff
@@ -1,15 +1,3 @@
-<% for (var i=0; i < ob.rating; i++) { %>
```

Comment:
I was going through the whole app and couldn't see any use for it. Didn't know OB supports partial stars. I'll put it back.